stubs/nanoid/nanoid/generate.pyi

```diff
@@ -0,0 +1,3 @@
+def generate(
+    alphabet: str = "_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ", size: int = 21  # noqa: Y053
```

Comment:
This (and in `stubs/nanoid/nanoid/non_secure_generate.pyi`) are the only time we noqa Y053 . Was there a specific reason we kept this string in particular ?

Reply:
I guess the chief point of Y053 is to catch very long strings that are dynamically generated by a function call at import time (so might be different on each machine) and/or are just too long to be useful. Here it's obviously not going to be different depending on which machine you're using. I guess whether it's useful or not to have this default value available in an IDE context is arguable

Reply:
It's not a major issue or anything, just wanted to make sure it was purposeful and not glanced over during review !